mmdeploy/apis/__init__.py

```diff
@@ -15,5 +15,6 @@
         'torch2onnx', 'torch2torchscript', 'build_task_processor',
         'get_predefined_partition_cfg', 'visualize_model'
     ]
-except Exception:
+except Exception as e:
+    print('Skip import apis, {}'.format(e))
```

Comment:
I found there is no warning log in the `mmdeploy.apis` try-exception block. Any reason why we do that? @grimoire 
Regarding the exception warning log, is there any concern about using `mmdeploy.utils.get_root_logger`?

Reply:
Sorry, I just forget.